ocaml/testsuite/tests/typing-layouts/mixed_records.ml

```diff
@@ -9,13 +9,7 @@ type t =
   }
```

Comment:
As with `mixed_constructor_arguments.ml`, I think we should set this test up to also run under beta and alpha.